src/build/index.ts

```diff
@@ -179,10 +179,11 @@ export async function build (input: string[] | Record<string,string>, opts: Buil
   
   // Make sure the rollup output array is in the same order as input array
   const inputObjKeys = Object.keys(inputObj);
-  output.sort((a, b) => inputObjKeys.indexOf(a.name) - inputObjKeys.indexOf(b.name));
+  const filteredOutput = output.filter(out => inputObjKeys.indexOf(out.name) !== -1);
```

Comment:
An easier way to do this filter would be:

```suggestion
  const filteredOutput = output.filter(out => out.isEntry);
```